queries.sql

```diff
@@ -8,3 +8,47 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
```

Comment:
- [ Required ] I think it would be nice if you make a `SELECT` query after performing the `ROLLBACK` to make sure that the changes you made in the `Transaction` are undone.

## Project requirements
![Screenshot from 2023-09-30 09-20-03](https://github.com/firepicaso/vet-clinic/assets/16392046/393a24dd-3ccf-45be-9250-f819afbd347c)